templates/participants/partials/participant_experiments.html

```diff
@@ -0,0 +1,66 @@
+{% load form_tags %}
+
+<table class="min-w-full divide-y divide-gray-300">
+  <thead>
+    <tr>
+      <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold">Experiment</th>
+      <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold">Joined on</th>
+      <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold">Last message received at</th>
+    </tr>
+  </thead>
+  <tbody class="divide-y divide-gray-200" x-data="{experimentsRow: null}">
+    {% for experiment, info in experiment_data.items %}
+      {% with sessions=info|dict_lookup:"sessions" participant_data=info|dict_lookup:"participant_data" %}
+        <tr
+          class="cursor-pointer"
+          x-bind:class="experimentsRow === {{ forloop.counter }} ? 'bg-gray-300' : ''"
+          x-on:click="experimentsRow = (experimentsRow === {{ forloop.counter }} ? null : {{ forloop.counter }})"
+        >
+          <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">{{ experiment.name }}</td>
+          <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">{{ experiment.joined_on }}</td>
+          <td class="px-3 py-4 text-sm text-gray-500">{{ experiment.last_message }}</td>
+        </tr>
+        <tr x-show="experimentsRow === {{ forloop.counter }}" x-cloak>
```

Comment:
maybe make this an accordion rather than a table?

https://v3.daisyui.com/components/accordion/#using-accordion-and-join-together

Reply:
We have to either go with a table or an accordian, since we can't really mix the two.

Here's what an accordian might look like
![image](https://github.com/dimagi/open-chat-studio/assets/64970009/b23fdf73-f024-4719-9bdb-bfff74a20aba)